### ISSUE TITLE
[lightbox-media-viewer - with carousel]: QA: Trap within the Light box modal dialog

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8910]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2519
- DPO consulting engineer: N/A

-----

### Description

Page element:
Light box modal 

Issue Description:
1. Screen reader focus is not trapped within the modal dialog.
2. When the focus is not trapped, there are couple of buttons that are identified by the screen reader just before and after the dialog with the label "start" and "end".

Expected:
1. Screen reader focus is expected to be trapped within the dialog in up/down arrow key along with tab/shift tab key
2. The visually hidden buttons should be hidden from SR as well

User Impact:
When the screen reader focus is not restricted within the modal dialog, it also identifies parent page content that disorients screen reader users.

### Component(s) impacted

lightbox-media-viewer - with carousel

### Browser

Firefox

### Carbon for IBM.com version

canary

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8757/iframe.html?id=components-lightbox-media-viewer--with-carousel

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
The test link is the deploy preview from #8757 but the focus issues were addressed in #9004.

The fixes should also be available in [canary](https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/iframe.html?id=components-lightbox-media-viewer--with-carousel)

**Comment by oliviaflory** [comment]
Not stale, issue in Review process

**Comment by proeung** [comment]
Closing this issue based on @andy-blum's comment (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/8910#issuecomment-1188302183). Feel free to re-open this issue if the bug still persists.
